application/src/main/java/org/opentripplanner/street/model/vertex/Vertex.java

```diff
@@ -161,6 +161,18 @@ public String getLabelString() {
     return getLabel().toString();
   }
 
+  public Vertex getParent() {
+    return this;
+  }
+
+  public String getMainLabelString() {
```

Comment:
This method is only used in tests. Can we find a way to keep it out of the production code?